app/Models/Flight.php

```diff
@@ -112,12 +112,12 @@ public function getIdentAttribute(): string
         $flight_id = $this->airline->code;
         $flight_id .= $this->flight_number;
 
-        if (filled($this->route_leg)) {
-            $flight_id .= '/L'.$this->route_leg;
+        if (filled($this->route_code)) {
+            $flight_id .= '-'.$this->route_code;
         }
 
-        if (filled($this->route_code)) {
-            $flight_id .= '/C'.$this->route_code;
+        if (filled($this->route_leg)) {
+            $flight_id .= '-'.$this->route_leg;
```

Comment:
I'm a little torn on this change. It's hard to tell what the code/leg are, especially if you need to enter into an ACARS app. I'll leave it for now but might change this later.

Reply:
normally code identifies the type of transport that the airplane makes ( for example X is Economy Class)
route leg is a numeric value that identifies the step in a long flight with multiple stops (for example a flight from San Francisco to Rome should stop in New York, Barcellona and Paris. So it mantains the same flight number but San Francisco-New York is leg 1, New York-Barcellona is leg 2, Barcellona-Paris is leg 3 and finally Paris-Rome is leg 4)

Reply:
The leg is correct, though I can't think of a time that I've seen the leg be used (at least with the airlines I've worked with. I write operations software for GE Aviation). RE the code - what you're talking about is the fare class; that's shown separately. The way those are coded at `[CLASS][CAPACITY or TOTAL]`, so "F10B20Y100" would be for that one. I have that noted in an issue to add that in.  For the flight itself, the code is actually  the revenue coding, which is only used internally for codeshares, etc. So it's not really important in terms of identifying flights, only the flight number and leg (though this is simplified too from actually operations)